app/views/episode_player/_form.html.erb

```diff
@@ -0,0 +1,64 @@
+<%= turbo_frame_tag "embed-player", data: {controller: "morphdom"} do %>
+
+  <div class="col-12 mb-4">
+    <div class="form-floating input-group" data-controller="dynamic-form">
+      <%= select_tag :embed_player_type, embed_player_type_options(embed_player_type), class: "form-control", data: {action: "dynamic-form#change"} %>
```

Comment:
Because I'm lazy, I used plain form tags instead of the formbuilder here.  But maybe I should have made this a slim-select as well...